opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -880,23 +880,23 @@ namespace Opm {
             const double tol_wells = param_.tolerance_wells_;
 
             const int nc = Opm::AutoDiffGrid::numCells(grid_);
-            const int np = numPhases();
 
             const auto& pv = geo_.poreVolume();
 
-            Vector R_sum(np);
-            Vector B_avg(np);
-            Vector maxCoeff(np);
-            Vector maxNormWell(np);
+            const int numComponent = numEq();
```

Comment:
shouldn't this plural? (`numComponents`)

also, `numEq()` is not necessarily the number of components anymore, so that variable name is probably quite confusing?